docs/release-notes/mongoid-8.1.txt

```diff
@@ -145,3 +145,40 @@ to or from a certain value:
   # => true
   person.name_changed?(from: "Trout", to: "Fletcher")
   # => false
+
+
+Added non-block form to ``#with``
+---------------------------------
+
+Mongoid 8.1 adds support for a non-block form for the ``#with`` method. The
+``#with`` method is used for changing the ``persistence context`` of a Model
+class, a document, and, starting in Mongoid 8.1, a criteria:
+
+.. code-block:: ruby
+
+  # The #with method on Model classes returns a criteria.
+  criteria = Band.with(database: "music-non-stop")
+  # => #<Mongoid::Criteria
+         selector: {}
+         options:  {}
+         class:    Band
+         embedded: false>
```

Comment:
perhaps it makes sense to change the Criteria#inspect method output to include the persistence context (if set)?